src/components/signup/SignupState.js

```diff
@@ -442,9 +444,13 @@ const Signup = ({ navigation }: { navigation: any, screenProps: any }) => {
   }, [navigation.state.index])
 
   const { scrollableContainer, contentContainer } = styles
-
   return (
-    <View style={{ flexGrow: shouldGrow ? 1 : 0 }}>
+    <View
+      style={{
+        flexGrow: shouldGrow ? 1 : 0,
+        minHeight: isShowKeyboard && getScreenHeight() <= 480 ? getScreenHeight() + 30 : getScreenHeight(),
```

Comment:
what is the logic here?

Reply:
we change component height because we need to move down button without cutting the button. only for small devices

Reply:
getScreenHeight would be the height with keyboard open, meaning smaller than real screen height. so how does it solve the issue?

Reply:
`getScreenHeight would be the height with keyboard open` - on IOS no. it is the height of the whole screen
we've added this only for ios.

Reply:
for ios we dont need minheight as screen size doesnt change with keyboard.
we need minheight for android so buttons will stay hidden

Reply:
this only for devices with height <= 480 (iPhone 5)
we need to increase min-height when the keyboard is shown. we need this to hide the button and other content behind the keyboard.

Reply:
but on iphone keyboard doesnt change height
also you only show here iphone how does it behave on android?
gif videos are again still too fast. reduce video resolution instead of speed. or make video slower but with less FPS

Reply:
on iphone the solution is scrollto on focus, on android you need minheight